tensilelite/Tensile/cmake/TensileConfig.cmake

```diff
@@ -209,7 +209,7 @@ function(TensileCreateLibraryFiles
     set(Options ${Options} "--build-id=${Tensile_BUILD_ID}")
   endif()
 
-  set(CommandLine ${VIRTUALENV_BIN_DIR}/${VIRTUALENV_PYTHON_EXENAME} ${Script} ${Options} ${Tensile_LOGIC_PATH} ${Tensile_OUTPUT_PATH} HIP)
+  set(CommandLine PYTHONPATH=$ENV{PYTHONPATH} ${VIRTUALENV_BIN_DIR}/${VIRTUALENV_PYTHON_EXENAME} ${Script} ${Options} ${Tensile_LOGIC_PATH} ${Tensile_OUTPUT_PATH} HIP)
```

Comment:
Prefer `${CMAKE_COMMAND} -E env PYTHONPATH=$ENV{PYTHONPATH} -- <<rest of command line>>`

That is more portable.

Reply:
Fixed